core/test/docs.toml

```diff
@@ -2,6 +2,7 @@
 # can also be set to a date-time like 1979-05-27T07:32:00
 starttime = 2019-01-01 # required
 endtime = 2021-01-01   # required
+crs = "EPSG:4326"      # required
```

Comment:
Maybe good to document what strings we support, just forwarding to the geopandas documentation page is good enough (indicating we can take proj strings, WKT etc).

Reply:
Done